operation/fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/config/ConfigurationAdapter.java

```diff
@@ -216,13 +216,11 @@
     String getCoreIamEndpoint();
 
     /**
-     * get the tx for the fast endpoint
-     *
-     * @implNote System value.
+     * get the tx for the fast chunk read during an sexport
```

Comment:
```suggestion
     * get the number ms to read payloads from the persistence layer before stopping to checkpoint
```
note: needs to be updated to say `seconds` if you take my recommendation below